### ISSUE TITLE
`<Pointout />` Content is not visible at 320px

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#564]
![image](https://user-images.githubusercontent.com/3180185/159587423-11bbd802-43c8-4f20-aad8-eaace5f05400.png)

<dl><dt>Description</dt><dd>Content or functionality is lost due to overlapping content when page width is adjusted to 320px.
At 320px content is not visible properly . Elements - 
1. With Position printout box
2. Change my position printout box</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
Content MUST be presented without loss of information or functionality when the viewport is set to 320 CSS pixels wide.

HOW TO FIX: 
Fix this issue by ensuring content does not overlap other content when the screen width is set to 320px by using techniques such as CSS media queries, grids, or flexbox to reflow content down to 320px width.

REFERENCE: 
Deque University:https://dequeuniversity.com/class/responsive-zoom2/zoom/linear-text-reflow
https://dequeuniversity.com/class/responsive-zoom2/responsive/
WCAG Understanding document:
https://www.w3.org/WAI/WCAG21/Understanding/reflow

BACKGROUND: 
People with low vision often read content on a desktop computer by scaling content using the browser zoom - up to 400%. When zooming causes content to overlap other content, users with low vision may not be able to see or use that content. When content is created so that it reflows without losing information or functionality at screen widths down to 320px, users can view a web page in a 1280px width browser screen and zoom the content to 400% (320px) without having to scroll side to side to read and without losing access to content.</dd><dt>Source code</dt><dd><pre><code><div class="Pointout__content"><h4 class="Pointout__heading">First time point out!</h4><p>This is a first time point out with a positioned pointout box</p></div></code></pre></dd><dt>Checkpoint</dt><dd>Reflow (1.4.10.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Pointout</dd></dl>

**Comment by scurker** [comment]
This component is getting deprecated/removed, see #900
